src/components/calcite-accordion/calcite-accordion.stories.js

```diff
@@ -0,0 +1,64 @@
+import { storiesOf } from "@storybook/html";
+import { withKnobs, select } from "@storybook/addon-knobs";
+import { darkBackground, parseReadme } from "../../../.storybook/helpers";
+import readme from "./readme.md";
+const notes = parseReadme(readme);
+
+storiesOf("Accordion", module)
+  .addDecorator(withKnobs)
+  .add(
+    "Accordion - Base",
+    () => `
+    <div style="width:300px;max-width:100%">
+    <calcite-accordion
+      theme="${select("theme", ["light", "dark"], "light")}",
```

Comment:
the reason I've been doing dark mode as a separate story is so that you can have it on a dark background. I think this will just still be on light gray and not very visible?